packages/react-devtools-extensions/src/background.js

```diff
@@ -182,5 +183,18 @@ chrome.runtime.onMessage.addListener((request, sender) => {
           break;
       }
     }
+  } else if (request.payload?.tabId) {
+    const tabId = request.payload?.tabId;
+    // This is sent from the devtools page when it is ready for injecting the backend
+    if (request.payload.type === 'react-devtools-inject-backend') {
```

Comment:
nit: Should we put `react-devtools-inject-backend` into constant?

I see its used in multiple places: 
https://github.com/facebook/react/blob/41b4714f195868193ddfb2a89d2373b1c61387f4/packages/react-devtools-extensions/src/main.js#L192
https://github.com/facebook/react/blob/41b4714f195868193ddfb2a89d2373b1c61387f4/packages/react-devtools-extensions/src/contentScripts/prepareInjection.js#L92

Reply:
Many event names/types are used in this way. I'm not against extracting them into a shared constant file as a better engineering work, but let's use another PR to take care of them all together.

Reply:
The big difference imo between using a constant and literals in that a constant will get allocated at the module level and would weight the sum of all the literals plus some small management extra. 
Literals on the other hand are only allocated when used and referenced (they may even get GCed like in `case constant`).